src/components/Text.tsx

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+
+export default function Text({ 
+  text = "",
+  italic = false, 
+  bold = false, 
+  underline = false, 
+  size = 'text-base', // uses tailwind utility class to set 1rem
+  color = 'text-black', 
```

Comment:
Also solved by above, but no explicit typing on props here as they come into the component